app/views/posts/show.html.erb

```diff
@@ -1 +1,27 @@
-<h1>This is the post by id file for testing</h1>
\ No newline at end of file
+<div class="container">
+  <div class="card mb-3" style="max-width: 800px;">
+    <div class="card-body text-success text-bg-light p-3">
+      <div class="d-flex justify-content-between">
+        <div>
+          <h4><span class="card-title">Post <%= @post_id %></span> <span>by <%= @post.author.name %></span></h4>
```

Comment:
👏 Great work on your project! One small suggestion for improvement:

- [ ] **Display Actual Post Title:** Currently, you're using placeholder text for each post like "Post". For a better user experience, it's recommended to display the actual title of each post. Update your code in the view to show the post title dynamically.
```suggestion
          <h4><span class="card-title"><%= @post.title %> <%= @post_id %></span> <span>by <%= @post.author.name %></span></h4>
```
By implementing this change, you'll provide users with more meaningful information about each post. Keep it up! 🚀👍